tests/epyccel/recognised_functions/test_numpy_funcs.py

```diff
@@ -2290,15 +2272,24 @@ def max_call(x : 'float[:]'):
     x = rand(10)
     assert(isclose(f1(x), max_call(x), rtol=RTOL, atol=ATOL))
 
-@pytest.mark.parametrize( 'language', (
-        pytest.param("fortran", marks = pytest.mark.fortran),
-        pytest.param("c", marks = [
-            pytest.mark.skip(reason="amax not implemented"),
-            pytest.mark.c]
-        ),
-        pytest.param("python", marks = pytest.mark.python)
-    )
-)
+def test_max_complex(language):
+    def max_call(x: 'complex128[:]'):
+        from numpy import amax
+        return amax(x)
+
+    f1 = epyccel(max_call, language=language)
+    x = randn(10) + 1j * randn(10)  # Generating an array of complex numbers
+    assert np.allclose(f1(x), max_call(x))
```

Comment:
Please add some tests where only the real part varies and only the imaginary part varies:

```suggestion
def test_max_complex(language):
    def max_call(x: 'complex128[:]'):
        from numpy import amax
        return amax(x)

    f1 = epyccel(max_call, language=language)
    x = randn(10) + 1j * randn(10)  # Generating an array of complex numbers
    assert np.allclose(f1(x), max_call(x))
    x = randn(10) + 1j  # Generating an array of complex numbers
    assert np.allclose(f1(x), max_call(x))
    x = 10 + 1j * randn(10)  # Generating an array of complex numbers
    assert np.allclose(f1(x), max_call(x))
```